src/main/java/com/google/api/gax/grpc/OperationFuture.java

```diff
@@ -50,9 +50,7 @@
 
 /** A RpcFuture which polls a service through OperationsApi for the completion of an operation. */
 public final class OperationFuture<ResponseT extends Message> extends AbstractRpcFuture<ResponseT> {
-  // TODO: Use exponential backoff in polling schedule
-  // https://github.com/googleapis/gax-java/issues/146
-  private static final Duration POLLING_INTERVAL = Duration.standardSeconds(1);
+  public static final Duration POLLING_INTERVAL = Duration.standardSeconds(1);
```

Comment:
I would say this should be renamed to `DEFAULT_POLLING_INTERVAL`, and maybe be package private to avoid exposing more than necessary.